### PR TITLE
[NSO-1384] Install nexkit and Magento2

### PR DESCRIPTION
**Pull request by aheadley**
This works, but needs more testing before review

### CODE REVIEW

File: playbooks/group_vars/all.yml [hunk 1]
```diff
@@ -1 +1,4 @@
 ---
+
+server_extra_packages:
+  - nexkit
```
Comment (clwells) [inline playbooks/group_vars/all.yml]: @aheadley - was thinking about this.  Should ImageMagick go here too?

**Comment by aheadley** [comment]
it cant, requires a repo not enabled by default

File: playbooks/tasks/install-app.yml [hunk 2]
```diff
@@ -1 +1,26 @@
 ---
+
+- set_fact:
+    siteworx_docroot: /home/{{ siteworx_user }}/{{ siteworx_domain }}/html
+
+- name: Check if Magento2 already installed
```
Comment (clwells) [inline playbooks/tasks/install-app.yml]: The install process from here needs to be rethought.  I will be passing a type indicator to let you know what app will be being installed from the following list:

https://github.com/clwells/core/blob/97edf29a49396faf3e4ee36f75264ccc0ca6be5e/nocworx/db/Migrations/20180225171259_nsd4857.php#L41-L47

I will also pass version info ('1' or '2' for example for magento).  Finally, I'll be passing admin user/pass info too and anything else that's needed by default (will need to know what this is).

**Comment by aheadley** [comment]
@clwells Please to re-review changes

**Comment by clwells** [comment]
Has this been tested and done what's expected on a staging and prod cloudhost?

**Comment by aheadley** [comment]
@clwells It has been tested on prod cloudhost, I'm not sure how to test on staging. Are there docs somewhere for that?

File: playbooks/tasks/install-app.yml [hunk 3]
```diff
@@ -1 +1,7 @@
 ---
+
+- set_fact:
+    siteworx_docroot: /home/{{ siteworx_user }}/{{ siteworx_domain }}/html
+
+- include: install-magento-2.yml
+  when: app_type == 'magento' and app_version == '2'
```
Comment (clwells) [inline playbooks/tasks/install-app.yml]: Can we add an install trigger here.  "and app_install"

Comment (clwells) [reply playbooks/tasks/install-app.yml]: NM - it's the install-app task so is implied.

**Pull request merged**